### PR TITLE
1913 take any

### PR DESCRIPTION
**Pull request by mcmajam**
### Changelog category (leave one):
- New Feature

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
... Add take_any and take_anyif aggregate functions for KQL.

> Information about CI checks: https://clickhouse.com/docs/en/development/continuous-integration/

### CODE REVIEW

File: src/Parsers/Kusto/KQL_ReleaseNote.md [hunk 1]
```diff
@@ -1304,4 +1304,6 @@ Please note that the functions listed below only take constant parameters for no
  - dcount()
  - dcountif()
  - bin
- 
\ No newline at end of file
+ - take_any()
+   Note: * is not currently a supported argument.
+ - take_anyif()
```
Comment (kashwy) [inline src/Parsers/Kusto/KQL_ReleaseNote.md:1309]: put to the top of doc, in the section of most recent release

File: src/Parsers/Kusto/KustoFunctions/KQLAggregationFunctions.cpp [hunk 2]
```diff
@@ -448,16 +448,41 @@ bool SumIf::convertImpl(String & out,IParser::Pos & pos)
...
+
+    if (fn_name.empty())
+        return false;
+    String expr;
+    ++pos;
+    while(pos->type != TokenType::ClosingRoundBracket)
+    {
+        if(pos->type != TokenType::Comma)
+        {
+            expr = expr + "any(" + String(pos->begin, pos->end) + ")";
```
Comment (kashwy) [inline src/Parsers/Kusto/KustoFunctions/KQLAggregationFunctions.cpp:461]: each argument need to be parsed by call getArgumet ( or getConvertedArgument) , think about this case:

`Customers| summarize take_any(strcat(FirstName,LastName))`

File: src/Parsers/Kusto/KQL_ReleaseNote.md (hunk 1 above)
Comment (mcmajam) [reply src/Parsers/Kusto/KQL_ReleaseNote.md]: Checked in correction.

File: src/Parsers/Kusto/KustoFunctions/KQLAggregationFunctions.cpp (hunk 2 above)
Comment (mcmajam) [reply src/Parsers/Kusto/KustoFunctions/KQLAggregationFunctions.cpp]: Checked in correction.

**Review by kashwy — APPROVED** [review]
run the code formatting, then push  and merge
check this about code formatting  :
https://github.ibm.com/ClickHouse/ClickHouse/wiki/Effective-use-of-Visual-Studio-Code#automatic-code-formatting

**Pull request merged**